src/unisparks/button.cpp

```diff
@@ -164,10 +164,11 @@ static const CRGB menuIconSpecial[ATOM_SCREEN_NUM_LEDS] = {
 
 CLEDController* atomMatrixScreenController = nullptr;
 
-void atomScreenDisplay() {
+void atomScreenDisplay(uint32_t currentMillis) {
   // M5Stack recommends not setting the atom screen brightness greater
   // than 20 to avoid melting the screen/cover over the LEDs.
-  atomMatrixScreenController->showLeds(20);
+  const uint32_t b = (currentMillis >> 6) & 0xF;
```

Comment:
nit: add comment explaining what this does (for example just copy the title of this pull request into a comment)

Reply:
Comment added